uber/forms/group.py

```diff
@@ -5,12 +5,16 @@
 from wtforms.validators import ValidationError
 
 from uber.config import c
-from uber.forms import AddressForm, CustomValidation, MultiCheckbox, MagForm, IntSelect, NumberInputGroup
+from uber.forms import AddressForm, CustomValidation, MultiCheckbox, MagForm, IntSelect, NumberInputGroup, Ranking
 from uber.forms.attendee import valid_cellphone
 from uber.custom_tags import format_currency, pluralize
 from uber.model_checks import invalid_phone_number
 
-__all__ = ['GroupInfo', 'ContactInfo', 'TableInfo', 'AdminGroupInfo', 'AdminTableInfo']
+__all__ = ['HotelLotteryApplication', 'GroupInfo', 'ContactInfo', 'TableInfo', 'AdminGroupInfo', 'AdminTableInfo']
+
+
+class HotelLotteryApplication(MagForm):
+    ranked_hotels = Ranking(c.HOTEL_LOTTERY.keys())
```

Comment:
Is Ranking() a widget or a field class? I see this form isn't being used, is it just left over from something else you were trying?